stellarphot/photometry/tests/fake_image.py

```diff
@@ -24,10 +25,46 @@ class FakeImage:
         the seed will be randomly generated.
```

Comment:
For consistency, please describe ALL the parameters including the new fwhm and n_repeats in the docstring.

Reply:
Fixed,